Auth/Platform/Auth.Platform.Shared/MyClass.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+namespace Auth.Platform.Shared
+{
+    public class MyClass
```

Comment:
Should we remove this if it's not being used for anything? Think this is just auto-generated

Reply:
Yep this will be removed